### PR TITLE
Add support for alternate instrumentation system in browser 

### PR DESCRIPTION
**Pull request by joelverhagen**
Progress on https://github.com/NuGet/Engineering/issues/3020

We will use this on nuget.org to swap out the instrumentation system with the Microsoft internal one that allows sending data to both Application Insights and to the internal endpoint which uses the "tenant key" configuration that I added.

### CODE REVIEW

File: src/NuGetGallery/Scripts/gallery/page-home.js [hunk 1]
```diff
@@ -116,8 +116,8 @@ $(function () {
         });
     }
 
-    function emitAIMetric(metricName) {
-        window.nuget.sendAiMetric(metricName, 1, {});
+    function emitMetric(metricName) {
```
Comment (loic-sharma) [inline src/NuGetGallery/Scripts/gallery/page-home.js:119]: Was renaming this function intentional? There's several references to `emitAIMetric` in this file

Comment (joelverhagen) [reply src/NuGetGallery/Scripts/gallery/page-home.js:119]: Yes, intentional. Good catch.

**Comment by zhhyu** [comment]
I am looking into it now, but it has been merged...

**Comment by joelverhagen** [comment]
> I am looking into it now, but it has been merged...

Apologies. Feel free to leave comments and I'll address them in a subsequent PR.

**Pull request merged**